analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/api/PackageVersionApi.java

```diff
@@ -29,24 +29,46 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/mvn/package_version")
+@RequestMapping("/package_version")
 public class PackageVersionApi {
 
 
     @GetMapping(value = "/{id}/rcg", produces = MediaType.APPLICATION_JSON_VALUE)
     ResponseEntity<String> getERCGLink(@PathVariable("id") long packageVersionId) {
+        String url;
         var artifact = KnowledgeBaseConnector.kbDao.getArtifactName(packageVersionId);
         if (artifact == null) {
             return new ResponseEntity<>("Package version not found", HttpStatus.NOT_FOUND);
         }
         var coordinate = artifact.split(Constants.mvnCoordinateSeparator);
-        var groupId = coordinate[0];
-        var artifactId = coordinate[1];
-        var version = coordinate[2];
-        String result = String.format("%smvn/%s/%s/%s_%s_%s.json", KnowledgeBaseConnector.rcgBaseUrl,
+        switch (KnowledgeBaseConnector.forge) {
+            case "mvn": {
+                var groupId = coordinate[0];
+                var artifactId = coordinate[1];
+                var version = coordinate[2];
+                url = String.format("%smvn/%s/%s/%s_%s_%s.json", KnowledgeBaseConnector.rcgBaseUrl,
                 artifactId.charAt(0), artifactId, artifactId, groupId, version);
-        result = result.replace("\\/", "/");
-        return new ResponseEntity<>(result, HttpStatus.OK);
+                break;
+            }
+            case "pypi": {
```

Comment:
There is a class with the supported forges by FASTEN, I suggest to use that instead of the hard-coded strings:
https://github.com/fasten-project/fasten/blob/develop/core/src/main/java/eu/fasten/core/data/Constants.java

Reply:
Thanks, changed this accordingly on faeb4ef